CHANGELOG.md

```diff
@@ -2,9 +2,34 @@
 
 All notable changes to this project will be documented in this file. Dates are displayed in UTC.
 
-Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
+#### [Unreleased](https://github.com/googlemaps/android-maps-utils/compare/1.0.0...HEAD)
 
-#### [Unreleased](https://github.com/googlemaps/android-maps-utils/compare/0.6.2...HEAD)
+#### [1.0.0](https://github.com/googlemaps/android-maps-utils/compare/0.6.2...1.0.0)
+
+> 24 February 2020
+
+### BREAKING CHANGES
+* Add MapObjectManager for multi-layer support with KML/GeoJSON renderers ([#380](https://github.com/googlemaps/android-maps-utils/pull/380))
+
+### Bug Fixes
+
+* **KmlLayer:** Update constructor to take in a Context instead ([#631](https://github.com/googlemaps/android-maps-utils/issues/631)) ([731f7e6](https://github.com/googlemaps/android-maps-utils/commit/731f7e61648ac1acd56770e0a841a172ade82847))
+* Maintain insertion order in clusters ([3ca965f](https://github.com/googlemaps/android-maps-utils/commit/3ca965f72c96fb44552c5410e6bdbc2815a82bac)), closes [#615](https://github.com/googlemaps/android-maps-utils/issues/615)
+* Make KML/GeoJSON lines and polygons clickable by default ([#614](https://github.com/googlemaps/android-maps-utils/issues/614)) ([122e96b](https://github.com/googlemaps/android-maps-utils/commit/122e96b8400af4c2f26f9c6237e3b4394b2a1d6a)), closes [#558](https://github.com/googlemaps/android-maps-utils/issues/558) [#454](https://github.com/googlemaps/android-maps-utils/issues/454) [#454](https://github.com/googlemaps/android-maps-utils/issues/454) [#558](https://github.com/googlemaps/android-maps-utils/issues/558)
```

Comment:
I just realized that #454 is linked twice here too

Reply:
...and also #558. looks like the tool just finds all the linked issues and spits it out in the changelog (#558 and #454 are mentioned twice).